calc.js

```diff
@@ -4,150 +4,217 @@ const keys = document.querySelector('#keyboard');
 const arithmeticKeys = document.querySelector('#arithmetic');
 const functionKeys = document.querySelector('#functions');
 
-keys.addEventListener( 'click' , getPress );
-arithmeticKeys.addEventListener( 'click' , getPress );
-functionKeys.addEventListener( 'click' , getPress );
+keys.addEventListener('click', getPress);
+arithmeticKeys.addEventListener('click', getPress);
+functionKeys.addEventListener('click', getPress);
 
 const obiekt = {
   value: 0,
   operator: '',
   secondValue: 0,
   state: ''
-} ;
+};
 
+const OPERATOR_PRESENT_KEY = 'operatorPresent'
```

Comment:
Hej, a dlaczego deklarujesz te stałe z dużej ? czy to jest jakiś inny typ danych ?

Reply:
Generalnie chodzi o zaznaczenie tego że jest to stała globalna która się nie zmieni w czasie działania programu bardziej niż inny typ danych to pewna konwencja nazywania zmiennych jaką zwykle przyjmuje pisząc program tak by inne osoby czytające na pierwszy rzut oka wiedziały do czego służy